pkgs/tools/networking/reaver-wps/parallel-build.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/Makefile.in	2012-01-18 10:02:40.000000000 +0100
```

Comment:
You'll probably need to apply the patch as well with something like `patches = [ ./parallel-build.patch ];`

Reply:
True, I lost that change when switching branches in git. Good you noticed. I'll change it right away.

Reply:
Now should be okay.